packages/react-frontend/src/MyApp.jsx

```diff
@@ -30,17 +30,17 @@ function MyApp() {
   }
 
   function checkUserLogin(person) {
-    return fetch(`${process.env.BACKEND_URL}/users/login`, {
+    return fetch(`${import.meta.env.VITE_BACKEND_URL}/users/login`, {
       method: "POST",
       headers: {
         "Content-Type": "application/json",
       },
       body: JSON.stringify(person),
```

Comment:
This needs to be changed to:
body: JSON.stringify({
        user_name: person.name,
        password: person.password
      }),

Reply:
Gotcha, done in Jonathan's branch